Source/Foundation/bsfUtility/Testing/BsTestCameraFlyer.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "BsPrerequisites.h"
+#include "Scene/BsComponent.h"
+#include "Math/BsMath.h"
+#include "Input/BsVirtualInput.h"
+
+namespace bs
+{
+	/** Component that controls movement and rotation of the scene object it's attached to. */
+	class CameraFlyer : public Component
```

Comment:
Don't like having this here. It's 1. duplicated and 2. it's an example-only class (for now, until we add a higher level gameplay layer).

Also not really sure about adding these kind of complex tests here (esp. not in the utility layer, which is too low level for this functionality). Adding them to examples might make more sense, but even that I'm not sure about. I haven't thought enough about these types of unit tests yet, I'd rather handle them in a non-unit tests (e.g. an example project) for now.